### PR TITLE
erro de injecao de controller

### PR DESCRIPTION
**Pull request by IsabellyCavalcante**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/julioleitao/sistemauto/pull/12" target="_blank">ReviewNinja

File: config/env.config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "development": {
         "secret": "YASFHP2Ynx9Uj2UnbKxq2eegODAlbzcCWh2Tlnjjpug=",
         "PORT": 8080,
-        "MONGO_URI": "mongodb://localhost/sistemauto"
+        "MONGO_URI": "mongodb://admin:admin@ds011890.mlab.com:11890/sistemauto"
```
Comment (julioleitao) [inline config/env.config.json]: desfazer

Comment (julioleitao) [inline config/env.config.json]: injeta o toast aqui

Comment (julioleitao) [inline config/env.config.json]: descomenta

Comment (IsabellyCavalcante) [inline config/env.config.json]: !fixed

Comment (IsabellyCavalcante) [inline config/env.config.json]: !resolve

Comment (IsabellyCavalcante) [inline config/env.config.json]: !resolved

**Pull request merged**